lib/matplotlib/path.py

```diff
@@ -17,7 +17,18 @@
 import matplotlib as mpl
 from . import _path, cbook
 from .cbook import _to_unmasked_float_array, simple_linear_interpolation
-from .bezier import split_bezier_intersecting_with_closedpath
+from .bezier import BezierSegment, split_bezier_intersecting_with_closedpath
+
+
+def _update_extents(extents, point):
```

Comment:
I'm not sure why I couldn't find a function like this in `transforms` already, but in the spirit of adding a little new API as possible I left this here.